### DISCUSSION TITLE
I had to apply for Student Benefits twice. Got approved twice, still has no access to benefits.

### DISCUSSION THREAD

**Discussion by kichuman28** [community/community#137313] [GitHub Education]
### Hi everyone,

So I first applied for the student benefits in my first year of college, wayy back in 2022. Wasn't aware of GitHub and it's benefits, just applied randomly. It got approved but I didn't follow up much on that. 

Fast forward to present, I'm trying to access GitHub Copilot via the Student Benefits Pack and I had to apply again, despite getting approved earlier on. It's been 3 days, I have no response. What do I do?

![image](https://github.com/user-attachments/assets/6ab9dfad-ac8e-45a7-9213-344db3c86005)
![image](https://github.com/user-attachments/assets/577af476-1664-4b28-9e02-13a879c8da64)

**Comment by davevad93** [comment]
Hi @kichuman28  ,

Student benefits last for 2 years, after which you have to reapply; your benefits approved in January 2022 have expired in January 2024,
so your current status is the one you submitted on August 28, 2024.

When benefits become available, the status label will turn purple (exactly like the one shown in your second screenshot) and change to:

- `Congrats! Your request for @kichuman28 was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; 

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
